queue_services/strr-pay/src/strr_pay/config.py

```diff
@@ -41,15 +41,15 @@
 """
 import os
 
-from dotenv import find_dotenv, load_dotenv
-
+from dotenv import find_dotenv
+from dotenv import load_dotenv
 
 # this will load all the envars from a .env file located in the project root (api)
 load_dotenv(find_dotenv())
 
 CONFIGURATION = {
     "development": "strr_pay.config.DevConfig",
-    "testing": "strr_pay.config.TestConfig",
+    "test": "strr_pay.config.TestConfig",
```

Comment:
if you check TestConfig definition https://github.com/bcgov/STRR/blob/e7dd5533c28d9421bcf00357a5113e77a56b3371/queue_services/strr-pay/src/strr_pay/config.py#L126 it sets TESTING = True, this variable is used by Flask and changes behavior of the app, e.g., error handling. This is appropriate for unit tests, renaming 'testing' to 'test' is OK, but using this configuration for gcp 'test' profile is not a good idea.

Reply:
okay studied the sample file you suggested at https://github.com/bcgov/bcros-common/blob/d9e58ac7168a4ef95953b7f760a0a777350028a3/notify-service/notify-api/src/notify_api/config.py#L177 and have made comparable changes.